doc/manual.xml

```diff
@@ -16,6 +16,7 @@
   <xi:include href="cross-compilation.xml" />
   <xi:include href="configuration.xml" />
   <xi:include href="functions.xml" />
+  <xi:include href="lib.xml" />
```

Comment:
@zimbatm yes, see my top comment. The last refactor did not modify the content at all.